src/servers/database.ts

```diff
@@ -465,7 +465,45 @@ export class DatabaseServer extends SkeletonServer {
       return null;
     }
   }
-
+  async eth_getTransactionsByBlockNumber(
+    _request: any,
+    blockNumber: web3.Quantity | web3.Tag
+  ): Promise<web3.TransactionResult | null> {
+    const blockNumber_ =
```

Comment:
Invalid block number handling in case of genesis block. Please have a look at this PR #122